src/main/webapp/components/workspace/workspace.js

```diff
@@ -236,7 +210,7 @@ export default () => {
                   })
                 }}
               />
-              <Results
+              <ResultIndexCards
                 results={results}
                 handleOpen={handleOpen}
```

Comment:
:pencil2: I don't think that workspace should care about the logic related to lists.

I think we should move the ResultListInteraction and Popover components as well as the selectedResult state into the ResultIndexCards component(if not it's own).